.travis.yml

```diff
@@ -3,15 +3,22 @@ language:
   - node_js
   - ruby
 node_js:
-  - 6
+  - node
+cache:
+  directories:
+    - ~/.npm
+    - ~/.cache
 install:
   - travis_retry gem install s3_website -v 3.4.0
-  - travis_retry npm install
+  - travis_retry npm ci
 before_script:
   - webpack
   - WEBPACK_TARGET=lib webpack
```

Comment:
Unrelated to this PR, but in reading up on Travis earlier today I came across a treatise on why build steps should be part of `script` rather than `before_script`. Unfortunately, I don't have the link handy, but the essence of it was that failures in `before_script` are considered the same as failures in `install`, i.e. they are reported like external failures that could potentially be resolved by simply trying again because maybe there was a network hiccup during the install process. Failures in the `script` phase are the only ones the are treated as build failures, but that's exactly how a webpack failure should be treated.